buildtest/config.py

```diff
@@ -33,16 +33,16 @@ def init():
         )
         os.mkdir(BUILDTEST_ROOT)
 
-    # Create subfolders for var and root
+    # Create subfolders for root, site
     create_dir(os.path.join(BUILDTEST_ROOT, "root"))
     create_dir(os.path.join(BUILDTEST_ROOT, "site"))
 
-    # Create config files, module files, and log file
-    create_config_file()
+    # Create settings files
+    create_settings_file()
 
 
-def check_configuration(config_path=None):
-    """Checks all keys in configuration file (settings/default.yml) are valid
+def check_settings(config_path=None):
+    """Checks all keys in configuration file (settings/settings.yml) are valid
```

Comment:
config_path maybe changed here too?

Reply:
yes that makes sense i will fix this in next commit